fetch.bs

```diff
@@ -3824,6 +3824,47 @@ the request.
  <var>request</var>'s <a for=request>origin</a> to  <var>request</var>'s <a for=request>client</a>'s
  <a for="environment settings object">origin</a>.
 
+ <li>
+  <p>If all of the following conditions are true:</p>
+
+  <ul class=brief>
+   <li><p><var>request</var>'s <a for=request>destination</a> is not "<code>document</code>"
```

Comment:
Should this use https://fetch.spec.whatwg.org/#navigation-request instead? The only overlap I can see with mode is for embed/object. Otherwise this would be redundant.

Reply:
Actually this line is wrong. It's ok to preload a document.